_data/versions.yml

```diff
@@ -1,7 +1,7 @@
 godot_3:
-  - name: "3.5.1"
+  - name: "3.5.2"
     flavor: "stable"
-    release_date: "28 September 2022"
+    release_date: "7 March 2023"
```

Comment:
The blog post is published today (8 March) but the release was technically tagged and uploaded on TF and GitHub yesterday (and that's what's in the CHANGELOG.md).

Doesn't matter much either way if we prefer to keep this one in sync with the blog announcement.

Reply:
So my idea here was to eventually list most releases here and perhaps have a page to download old releases instead of making people go to TuxFamily directly.

For that reason, you can simply add the new record above the existing one here, it should keep working, and keep the information about the old version stored as well.

Reply:
Makes sense, fixed.